src/EnergyPlus/DataSizing.hh

```diff
@@ -925,7 +932,7 @@ namespace DataSizing {
         Real64 FractionOfAutosizedHeatingCapacity; // fraction of of heating total capacity
         Real64 CoolingTotalCapacity;               // system total cooling capacity
         Real64 HeatingTotalCapacity;               // system total heating capacity
-        int CoolingPeakLoadType;                   // Type of peak to size cooling coils on   1=SensibleCoolingLoad; 2=TotalCooligLoad
+        PeakLoad peakLoad;                         // Type of peak to size cooling coils on SensibleCooling or TotalCooling
```

Comment:
The old variable name is much more informative. `peakLoad` implies it's the load in W.